scripts-available/CDB_Groups_API.sql

```diff
@@ -197,19 +197,30 @@ CREATE OR REPLACE
 FUNCTION @extschema@._CDB_Group_API_Auth(username text, password text)
     RETURNS TEXT AS
 $$
+    import sys
     import base64
-    return base64.encodestring('%s:%s' % (username, password)).replace('\n', '')
+
+    data_to_encode = '%s:%s' % (username, password)
+    if sys.version_info[0] < 3:
+        data_encoded = base64.encodestring(data_to_encode)
+    else:
+        data_encoded = base64.b64encode(data_to_encode.encode()).decode()
+
+    data_encoded = data_encoded.replace('\n', '')
+    return data_encoded
 $$ LANGUAGE '@@plpythonu@@' VOLATILE PARALLEL UNSAFE;
 
 -- url must contain a '%s' placeholder that will be replaced by current_database, for security reasons.
 CREATE OR REPLACE
 FUNCTION @extschema@._CDB_Group_API_Request(method text, url text, body text, valid_return_codes int[])
     RETURNS int AS
 $$
+    python_v2 = True
     try:
-        import httplib as client
+      import httplib as client
     except:
-        from http import client
+      from http import client
+      python_v3 = False
```

Comment:
Name should be python_v2 here

Reply:
:man_facepalming: I changed it in cartodb/lib/sql/scripts-available instead of here. Sorry about that and good catch!